src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -141,6 +147,17 @@ const createIdpButtons = (remediations) => {
   });
 };
 
+const createPIVButton = (idpObject) => {
+  return {
+    className: 'piv-poc piv-button',
+    title: 'PIV POC Button',
+    click: (e) => {
```

Comment:
what's difference with declare `href` ?

Reply:
no difference, this is just a POC. ultimately we will use click to render a new PIV view